copilot/data-store-celery/manifest.yml

```diff
@@ -0,0 +1,52 @@
+# The manifest for the "data-store-celery" service.
+# Read the full specification for the "Backend Service" type at:
+#  https://aws.github.io/copilot-cli/docs/manifest/backend-service/
+
+name: data-store-celery
+type: Backend Service
+
+image:
+  healthcheck:
+    command: ["CMD-SHELL", "launcher celery -A app.celery_app inspect ping"]
+    interval: 30s
+    retries: 2
+    timeout: 15s
+    start_period: 30s
+
+# TODO: Sort out log level and logging in Python using FSD_LOG_LEVEL; make sure logs are emitted in JSON.
+entrypoint: launcher celery -A app.celery_app worker --loglevel=INFO
+
+cpu: 2048
+memory: 4096
+platform: linux/x86_64 # See https://aws.github.io/copilot-cli/docs/manifest/backend-service/#platform
+count: 1
+exec: true
+network:
+  # Enable Service Connect for intra-environment traffic between services.
+  connect: true
+
+variables:
+  FLASK_ENV: ${COPILOT_ENVIRONMENT_NAME}
+  # Sentry DSN is OK to be public see: https://docs.sentry.io/product/sentry-basics/dsn-explainer/#dsn-utilization
+  SENTRY_DSN: https://6a2623e302e641ba88eabe6675a70ddf@o1432034.ingest.sentry.io/4505390859747328
+
+secrets:
+  NOTIFY_FIND_API_KEY: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/NOTIFY_FIND_API_KEY
+  REDIS_URL:
+    from_cfn: ${COPILOT_APPLICATION_NAME}-${COPILOT_ENVIRONMENT_NAME}-DataStoreRedisUrl
+  DATASTORECLUSTER_SECRET:
+    from_cfn: ${COPILOT_APPLICATION_NAME}-${COPILOT_ENVIRONMENT_NAME}-DataStoreClusterSecret
+
+environments:
+  production:
+    count: 2
```

Comment:
Maybe we can get away with one instance here as 0 downtime is not actually needed for a worker?

Reply:
Good catch - I forgot to remove this. Have also just set the trace count to 1 in prod, as the number of downloads is so small I'd be curious to get stats on them all (initially at least)...